models/_sector/depin/geodnet/geodnet_revenue.sql

```diff
@@ -0,0 +1,127 @@
+{{ config(
+    schema = 'geodnet'
+    , alias = 'revenue'
+    , materialized = 'incremental'
+    , file_format = 'delta'
+    , incremental_strategy = 'merge'
+    , unique_key = ['date', 'chain', 'name']
```

Comment:
be sure to update `chain` to `blockchain` here, as well as throughout schema yml file